packages/api-bindings/src/graphql/profile.graphql

```diff
@@ -141,3 +145,49 @@ query MutualFollowersProfiles(
     }
   }
 }
+
+mutation CreateSetProfileMetadataTypedData(
+  $request: CreatePublicSetProfileMetadataURIRequest!
+  $options: TypedDataOptions
+) {
+  result: createSetProfileMetadataTypedData(request: $request, options: $options) {
+    id
+    expiresAt
+    typedData {
+      types {
+        SetProfileMetadataURIWithSig {
+          name
+          type
+        }
+      }
+      domain {
+        name
+        chainId
+        version
+        verifyingContract
+      }
+      value {
+        nonce
+        deadline
+        profileId
+        metadata
+      }
+    }
+  }
+}
+
+mutation CreateSetProfileMetadataViaDispatcher(
+  $request: CreatePublicSetProfileMetadataURIRequest!
+) {
+  result: createSetProfileMetadataViaDispatcher(request: $request) {
+    __typename
+
+    ... on RelayerResult {
+      ...RelayerResult
+    }
+
+    ... on RelayError {
+      ...RelayError
+    }
+  }
+}
```

Comment:
These 2 mutations are strictly not related to the changes in this PR but part of the same feature. They will be used in the follow PR.